docs/source/starters/new_project_tools.md

```diff
@@ -1,4 +1,229 @@
+
+You can skip the step to name the project by adding it to your command. For example:
+
+```bash
+kedro new --name=spaceflights
+```
+
+### Tools
+You are then asked to select which tools to include. Choose from the list using comma separated values `(1,2,4)`, a range of values `(3-5)`, or the key words `all` or `none`. Skipping the prompt by entering no value will result in the default selection of `none`. Further information about each of the tools is described below in [Kedro tools](#kedro-tools).
+
+
+| ![](../meta/images/kedro-new-tools-prompt.png)|
+|:--:|
+| *Prompt for tools selection* |
+
+
+You may also specify your tools selection directly from the command line by using the flag `--tools`:
+
+```bash
+kedro new --tools=<your tool selection>
+```
+
+To specify your desired tools you must provide them by name as a comma separated list, for example `--tools=lint,test,viz`. The following tools are available for selection: `lint`, `test`, `log`, `docs`, `data`, `pyspark`, and `viz`.
+
+A list of available tools can also be accessed by running `kedro new --help`
+
+![](../meta/images/kedro-new-help.png)
+
+
+### Example code
+In the final step you are asked whether you want to populate the project with an example pipeline. Further information about this pipeline is below in [Example pipeline](#example-pipeline).
+
+You can add the example pipeline to your new project as follows:
+
+```bash
+kedro new --example=y
+```
+
+## `kedro new --config=`
+
+You can also supply values to `kedro new` by providing a YML configuration file to your `kedro new` command. Consider the following file:
+
+```yaml
+# config.yml
+
+"project_name":
+    "My Project"
+
+"repo_name":
+    "my-project"
+
+"python_package":
+    "my project"
+
+"tools":
+    "2-6"
+```
+
+To create a new project using the file to supply details to `kedro new`, run the following command:
+
+```bash
+kedro new --config=<path/to/config.yml>
+```
+
+``` {note}
+Note: When using a configuration file to create a new project, you must provide values for the project name, repository name, and package names.
+```
+
+Specifying your tools selection is optional, omitting them results in the default selection of `none`.
+
+<!--TO DO Need also to add examples?? -->
+
+## Kedro tools
+
+Tools are [modular functionalities that can be added to a basic Kedro project] . They allow for [...]. When creating a new project, you may select one or more of the available tools, or none at all.
+
+The available tools include: linting, testing, custom logging, documentation, data structure, PySpark, and Kedro-Viz.
+
+### Linting
+
+The Kedro linting tool introduces [`black`](https://black.readthedocs.io/en/stable/index.html) and [`ruff`](https://docs.astral.sh/ruff/) as dependencies in your new project's requirements. After project creation, make sure these are installed by running the following command:
+
+```bash
+pip install -r path/to/project/root/requirements.txt
```

Comment:
Do we need to prefix all `pip install -r` instructions with `path/to/project/root`? Alternatively we can assume that the user is already there (or explicitly say it)